server/notification-providers/send-grid.js

```diff
@@ -0,0 +1,65 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class SendGrid extends NotificationProvider {
+  name = "SendGrid";
```

Comment:
please fix the lining issues in this file

Reply:
done